pwa/static/themes/rotterdam.css

```diff
@@ -44,6 +44,8 @@
   --web-app-size-3xl: var(--skeleton-size-3xl);
   --web-app-size-4xl: var(--skeleton-size-4xl);
 
+  --web-app-max-width-400: 400px;
```

Comment:
These tokens can exist, however, I think they should be more descriptive (as stated in other feedback), and I think they should be used within the _context_ and not within the component itself.